const data = [
  {
    'abstract':
      'In this paper, we present a Convolutional Neural Network (CNN) regression approach to address the two major limitations of existing intensity-based 2-D/3-D registration technology: 1) slow computation and 2) small capture range. Different from optimization-based methods, which iteratively optimize the transformation parameters over a scalar-valued metric function representing the quality of the registration, the proposed method exploits the information embedded in the appearances of the digitally reconstructed radiograph and X-ray images, and employs CNN regressors to directly estimate the transformation parameters. An automatic feature extraction step is introduced to calculate 3-D pose-indexed features that are sensitive to the variables to be regressed while robust to other factors. The CNN regressors are then trained for local zones and applied in a hierarchical manner to break down the complex regression task into multiple simpler sub-tasks that can be learned separately. Weight sharing is furthermore employed in the CNN regression model to reduce the memory footprint. The proposed approach has been quantitatively evaluated on 3 potential clinical applications, demonstrating its significant advantage in providing highly accurate real-time 2-D/3-D registration with a significantly enlarged capture range when compared to intensity-based methods.',
    'authors': 'S. Miao,  Z. J. Wang,  R. Liao',
    'description': 'IEEE Transactions on Medical Imaging. 2016',
    'doi': '10.1109/TMI.2016.2521800',
    'duplicate': false,
    'email': ['smiao@ece.ubc.ca', 'zjwang@ece.ubc.ca', 'rui.liao@siemens.com'],
    'email_new': [
      'smiao@ece.ubc.ca',
      'zjwang@ece.ubc.ca',
      'rui.liao@siemens.com',
    ],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7393571',
    'key': '44acb4cb-d1e0-465f-abc2-9441707db2d3',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title': 'a cnn regression approach for real-time 2d/3d registration',
    'tools': ['caffe'],
    'year': 2016.0,
    'id': '44acb4cb-d1e0-465f-abc2-9441707db2d3.pdf',
    'use': 1.0,
    'domain': 'radiology',
    'task_1': 'registration',
    'task_2': '0',
    'data_type': 'xray',
    'location': 'multi-organ',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '0',
    'training': 0.0,
    'validation': 0.0,
    'testing': 0.0,
    'cross validation': 0.0,
    'country_1': 'USA',
    'country_2': 'Canada',
    'country_3': 0.0,
    'journal': 'IEEE Transactions on Medical Imaging',
    'impact_factor': 3.942,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Nuclear segmentation in digital microscopic tissue images can enable extraction of high-quality features for nuclear morphometrics and other analysis in computational pathology. Conventional image processing techniques, such as Otsu thresholding and watershed segmentation, do not work effectively on challenging cases, such as chromatin-sparse and crowded nuclei. In contrast, machine learning-based segmentation can generalize across various nuclear appearances. However, training machine learning algorithms requires data sets of images, in which a vast number of nuclei have been annotated. Publicly accessible and annotated data sets, along with widely agreed upon metrics to compare techniques, have catalyzed tremendous innovation and progress on other image classification problems, particularly in object recognition. Inspired by their success, we introduce a large publicly accessible data set of hematoxylin and eosin (H&E)-stained tissue images with more than 21000 painstakingly annotated nuclear boundaries, whose quality was validated by a medical doctor. Because our data set is taken from multiple hospitals and includes a diversity of nuclear appearances from several patients, disease states, and organs, techniques trained on it are likely to generalize well and work right out-of-the-box on other H&E-stained images. We also propose a new metric to evaluate nuclear segmentation results that penalizes object- and pixel-level errors in a unified manner, unlike previous metrics that penalize only one type of error. We also propose a segmentation technique based on deep learning that lays a special emphasis on identifying the nuclear boundaries, including those between the touching or overlapping nuclei, and works well on a diverse set of test images.',
    'authors':
      'N. Kumar,  R. Verma,  S. Sharma,  S. Bhargava,  A. Vahadane,  A. Sethi',
    'description': 'IEEE Transactions on Medical Imaging. 2017',
    'doi': '10.1109/TMI.2017.2677499',
    'duplicate': false,
    'email': ['neeraj.kumar.iitg@gmail.com', 'amitsethi@gmail.com'],
    'email_new': ['neeraj.kumar.iitg@gmail.com', 'amitsethi@gmail.com'],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7872382',
    'key': '6011dff4-4995-40e3-9827-068f13a3b4e1',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title':
      'a dataset and a technique for generalized nuclear segmentation for computational pathology',
    'tools': ['torch'],
    'year': 2017.0,
    'id': '6011dff4-4995-40e3-9827-068f13a3b4e1.pdf',
    'use': 1.0,
    'domain': 'pathology',
    'task_1': 'segmentation',
    'task_2': '0',
    'data_type': 'H&E stained',
    'location': 'cell',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 1.0,
    'data_link_1': 'http://nucleisegmentationbenchmark.weebly.com/',
    'data_link_2': '0',
    'data_ok': 1.0,
    'code_public': 1.0,
    'code_link': 'https://github.com/neerajkumarvaid/NucleiSegmentation',
    'code_ok': 1.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': null,
    'training': null,
    'validation': null,
    'testing': null,
    'cross validation': 0.0,
    'country_1': 'India',
    'country_2': '0',
    'country_3': 0.0,
    'journal': 'IEEE Transactions on Medical Imaging',
    'impact_factor': 3.942,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Wireless Capsule Endoscopy (WCE) is a standard non-invasive modality for small bowel examination. Recently, the development of computer-aided diagnosis (CAD) systems for gastrointestinal (GI) bleeding detection in WCE image videos has become an active research area with the goal of relieving the workload of physicians. Existing methods based primarily on handcrafted features usually give insufficient accuracy for bleeding detection, due to their limited capability of feature representation. In this paper, we present a new automatic bleeding detection strategy based on a deep convolutional neural network and evaluate our method on an expanded dataset of 10,000 WCE images. Experimental results with an increase of around 2 percentage points in the Fi score demonstrate that our method outperforms the state-of-the-art approaches in WCE bleeding detection. The achieved Fi score is of up to 0.9955.',
    'authors': 'Xiao Jia, Meng MQ.',
    'description': 'Conf Proc IEEE Eng Med Biol Soc.  2016',
    'doi': null,
    'duplicate': false,
    'email': ['xjia@ee.cuhk.edu.hk', 'qhmeng@ee.cuhk.edu.hk'],
    'email_new': ['xjia@ee.cuhk.edu.hk', 'qhmeng@ee.cuhk.edu.hk'],
    'fullUrl': 'https://www.ncbi.nlm.nih.gov//pubmed/28268409',
    'key': '19fe9d15-1a95-44c4-be60-812ccfb9437c',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'pubmed',
    'title':
      'a deep convolutional neural network for bleeding detection in wireless capsule endoscopy images',
    'tools': ['caffe'],
    'year': 2016.0,
    'id': '19fe9d15-1a95-44c4-be60-812ccfb9437c.pdf',
    'use': 1.0,
    'domain': 'endoscopy',
    'task_1': 'detection',
    'task_2': '0',
    'data_type': 'wireless capsule endoscopy',
    'location': 'gastrointestinal',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '10000',
    'training': 8200.0,
    'validation': 0.0,
    'testing': 1800.0,
    'cross validation': 0.0,
    'country_1': 'Hong Kong',
    'country_2': '0',
    'country_3': 0.0,
    'journal': '0',
    'impact_factor': 0.0,
    'conference':
      '2016 38th Annual International Conference of the IEEE Engineering in Medicine and Biology Society',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Breast lesion detection using ultrasound imaging is considered an important step of Computer-Aided Diagnosis systems. Over the past decade, researchers have demonstrated the possibilities to automate the initial lesion detection. However, the lack of a common dataset impedes research when comparing the performance of such algorithms. This paper proposes the use of deep learning approaches for breast ultrasound lesion detection and investigates three different methods: a Patch-based LeNet, a U-Net, and a transfer learning approach with a pretrained FCN-AlexNet. Their performance is compared against four state-of-the-art lesion detection algorithms (i.e. Radial Gradient Index, Multifractal Filtering, Rule-based Region Ranking and Deformable Part Models). In addition, this paper compares and contrasts two conventional ultrasound image datasets acquired from two different ultrasound systems. Dataset A comprises 306 (60 malignant and 246 benign) images and Dataset B comprises 163 (53 malignant and 110 benign) images. To overcome the lack of public datasets in this domain, Dataset B will be made available for research purposes. The results demonstrate an overall improvement by the deep learning approaches when assessed on both datasets in terms of True Positive Fraction, False Positives per image, and F-measure.',
    'authors':
      'Yap MH, Pons G, Marti J, Ganau S, Sentis M, Zwiggelaar R, Davison AK, Marti R.',
    'description': 'IEEE J Biomed Health Inform.  2017',
    'doi': null,
    'duplicate': false,
    'email': [
      'm.yap@mmu.ac.uk',
      'gponsro@uoc.edu',
      'robert.marti@udg.edu',
      'MSentis@tauli.cat',
      'rrz@aber.ac.uk',
      'adrian.davison@manchester.ac.uk',
    ],
    'email_new': [
      'm.yap@mmu.ac.uk',
      'gponsro@uoc.edu',
      'robert.marti@udg.edu',
      'MSentis@tauli.cat',
      'rrz@aber.ac.uk',
      'adrian.davison@manchester.ac.uk',
    ],
    'fullUrl': 'https://www.ncbi.nlm.nih.gov//pubmed/28796627',
    'key': '211d576b-be6a-4ba2-bded-41d6348174e4',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'pubmed',
    'title':
      'automated breast ultrasound lesions detection using convolutional neural networks',
    'tools': ['caffe'],
    'year': 2017.0,
    'id': '211d576b-be6a-4ba2-bded-41d6348174e4.pdf',
    'use': 1.0,
    'domain': 'radiology',
    'task_1': 'detection',
    'task_2': '0',
    'data_type': 'us',
    'location': 'breast',
    'method_1': 'cnn',
    'method_2': 'fcn',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 1.0,
    'data_link_1': 'http://www2.docm.mmu.ac.uk/STAFF/m.yap/dataset.php',
    'data_link_2': '0',
    'data_ok': 1.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1':
      'https://lmb.informatik.uni-freiburg.de/resources/opensource/unet.en.html',
    'based_on_2': '0',
    'transfer_learning': 1.0,
    'total': '469',
    'training': 0.0,
    'validation': 0.0,
    'testing': 0.0,
    'cross validation': 1.0,
    'country_1': 'UK',
    'country_2': 'Spain',
    'country_3': 0.0,
    'journal': 'IEEE Journal of Biomedical and Health Informatics',
    'impact_factor': 3.451,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': '*',
  },
  {
    'abstract':
      'Automated melanoma recognition in dermoscopy images is a very challenging task due to the low contrast of skin lesions, the huge intraclass variation of melanomas, the high degree of visual similarity between melanoma and non-melanoma lesions, and the existence of many artifacts in the image. In order to meet these challenges, we propose a novel method for melanoma recognition by leveraging very deep convolutional neural networks (CNNs). Compared with existing methods employing either low-level hand-crafted features or CNNs with shallower architectures, our substantially deeper networks (more than 50 layers) can acquire richer and more discriminative features for more accurate recognition. To take full advantage of very deep networks, we propose a set of schemes to ensure effective training and learning under limited training data. First, we apply the residual learning to cope with the degradation and overfitting problems when a network goes deeper. This technique can ensure that our networks benefit from the performance gains achieved by increasing network depth. Then, we construct a fully convolutional residual network (FCRN) for accurate skin lesion segmentation, and further enhance its capability by incorporating a multi-scale contextual information integration scheme. Finally, we seamlessly integrate the proposed FCRN (for segmentation) and other very deep residual networks (for classification) to form a two-stage framework. This framework enables the classification network to extract more representative and specific features based on segmented results instead of the whole dermoscopy images, further alleviating the insufficiency of training data. The proposed framework is extensively evaluated on ISBI 2016 Skin Lesion Analysis Towards Melanoma Detection Challenge dataset. Experimental results demonstrate the significant performance gains of the proposed framework, ranking the first in classification and the second in segmentation among 25 teams and 28 teams, r- spectively. This study corroborates that very deep CNNs with effective training mechanisms can be employed to solve complicated medical image analysis tasks, even with limited training data.',
    'authors': 'L. Yu,  H. Chen,  Q. Dou,  J. Qin,  P. A. Heng',
    'description': 'IEEE Transactions on Medical Imaging. 2017',
    'doi': '10.1109/TMI.2016.2642839',
    'duplicate': false,
    'email': [
      'lqyu@cse.cuhk.edu.hk',
      'hchen@cse.cuhk.edu.hk',
      'qdou@cse.cuhk.edu.hk',
      'pheng@cse.cuhk.edu.hk',
      'harry.qin@polyu.edu.hk',
    ],
    'email_new': [
      'lqyu@cse.cuhk.edu.hk',
      'hchen@cse.cuhk.edu.hk',
      'qdou@cse.cuhk.edu.hk',
      'pheng@cse.cuhk.edu.hk',
      'harry.qin@polyu.edu.hk',
    ],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7792699',
    'key': '0104bacd-b252-4612-b689-cc1c856707b8',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title':
      'automated melanoma recognition in dermoscopy images via very deep residual networks',
    'tools': ['caffe'],
    'year': 2017.0,
    'id': '0104bacd-b252-4612-b689-cc1c856707b8.pdf',
    'use': 1.0,
    'domain': 'dermatology',
    'task_1': 'segmentation',
    'task_2': 'classification',
    'data_type': 'dermoscopy',
    'location': 'skin',
    'method_1': 'fcrn',
    'method_2': 'cnn',
    'method_3': '0',
    'data_public': 1.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': 'https://isic-archive.com/',
    'data_link_2': '0',
    'data_ok': 1.0,
    'code_public': 1.0,
    'code_link': 'https://github.com/yulequan/melanoma-recognition',
    'code_ok': 1.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '1250',
    'training': 900.0,
    'validation': 0.0,
    'testing': 350.0,
    'cross validation': 0.0,
    'country_1': 'Hong Kong',
    'country_2': '0',
    'country_3': 0.0,
    'journal': 'IEEE TRANSACTIONS ON MEDICAL IMAGING',
    'impact_factor': 3.942,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Colorectal cancer (CRC) is a leading cause of cancer deaths worldwide. Although polypectomy at early stage reduces CRC incidence, 90% of the polyps are small and diminutive, where removal of them poses risks to patients that may outweigh the benefits. Correctly detecting and predicting polyp type during colonoscopy allows endoscopists to resect and discard the tissue without submitting it for histology, saving time, and costs. Nevertheless, human visual observation of early stage polyps varies. Therefore, this paper aims at developing a fully automatic algorithm to detect and classify hyperplastic and adenomatous colorectal polyps. Adenomatous polyps should be removed, whereas distal diminutive hyperplastic polyps are considered clinically insignificant and may be left in situ . A novel transfer learning application is proposed utilizing features learned from big nonmedical datasets with 1.4-2.5 million images using deep convolutional neural network. The endoscopic images we collected for experiment were taken under random lighting conditions, zooming and optical magnification, including 1104 endoscopic nonpolyp images taken under both white-light and narrowband imaging (NBI) endoscopy and 826 NBI endoscopic polyp images, of which 263 images were hyperplasia and 563 were adenoma as confirmed by histology. The proposed method identified polyp images from nonpolyp images in the beginning followed by predicting the polyp histology. When compared with visual inspection by endoscopists, the results of this study show that the proposed method has similar precision (87.3% versus 86.4%) but a higher recall rate (87.6% versus 77.0%) and a higher accuracy (85.9% versus 74.3%). In conclusion, automatic algorithms can assist endoscopists in identifying polyps that are adenomatous but have been incorrectly judged as hyperplasia and, therefore, enable timely resection of these polyps at an early stage before they develop into invasive cancer.',
    'authors': 'Zhang R, Zheng Y, Mak TW, Yu R, Wong SH, Lau JY, Poon CC.',
    'description': 'IEEE J Biomed Health Inform.  2017',
    'doi': null,
    'duplicate': false,
    'email': ['cpoon@surgery.cuhk.edu.hk'],
    'email_new': ['cpoon@surgery.cuhk.edu.hk'],
    'fullUrl': 'https://www.ncbi.nlm.nih.gov//pubmed/28114040',
    'key': '3bd582af-d98e-4f9e-8985-040bca4e4925',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'pubmed',
    'title':
      'automatic detection and classification of colorectal polyps by transferring low-level cnn features from nonmedical domain',
    'tools': ['caffe'],
    'year': 2017.0,
    'id': '3bd582af-d98e-4f9e-8985-040bca4e4925.pdf',
    'use': 1.0,
    'domain': 'endoscopy',
    'task_1': 'detection',
    'task_2': 'classification',
    'data_type': 'colonoscopy',
    'location': 'colon',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 1.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': 'http://www.depeca.uah.es/colonoscopy_dataset/',
    'data_link_2': '0',
    'data_ok': 1.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1':
      'https://github.com/BVLC/caffe/tree/master/models/bvlc_reference_caffenet',
    'based_on_2': '0',
    'transfer_learning': 1.0,
    'total': '291',
    'training': 0.0,
    'validation': 0.0,
    'testing': 0.0,
    'cross validation': 1.0,
    'country_1': 'Hong Kong',
    'country_2': '0',
    'country_3': 0.0,
    'journal': 'IEEE Journal of Biomedical and Health Informatics',
    'impact_factor': 3.451,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Echocardiography (echo) is a skilled technical procedure that depends on the experience of the operator. The aim of this paper is to reduce user variability in data acquisition by automatically computing a score of echo quality for operator feedback. To do this, a deep convolutional neural network model, trained on a large set of samples, was developed for scoring apical four-chamber (A4C) echo. In this paper, 6,916 end-systolic echo images were manually studied by an expert cardiologist and were assigned a score between 0 (not acceptable) and 5 (excellent). The images were divided into two independent training-validation and test sets. The network architecture and its parameters were based on the stochastic approach of the particle swarm optimization on the training-validation data. The mean absolute error between the scores from the ultimately trained model and the expert\'s manual scores was 0.71 \u00b1 0.58. The reported error was comparable to the measured intra-rater reliability. The learned features of the network were visually interpretable and could be mapped to the anatomy of the heart in the A4C echo, giving confidence in the training result. The computation time for the proposed network architecture, running on a graphics processing unit, was less than 10 ms per frame, sufficient for real-time deployment. The proposed approach has the potential to facilitate the widespread use of echo at the point-of-care and enable early and timely diagnosis and treatment. Finally, the approach did not use any specific assumptions about the A4C echo, so it could be generalizable to other standard echo views.',
    'authors':
      'Abdi AH, Luong C, Tsang T, Allan G, Nouranian S, Jue J, Hawley D, Fleming S, Gin K, Swift J, Rohling R, Abolmaesumi P.',
    'description': 'IEEE Trans Med Imaging.  2017',
    'doi': null,
    'duplicate': false,
    'email': ['purang@ece.ubc.ca'],
    'email_new': ['purang@ece.ubc.ca'],
    'fullUrl': 'https://www.ncbi.nlm.nih.gov//pubmed/28391191',
    'key': '3db1dca1-3ee6-416e-ac50-74359d00efb7',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'pubmed',
    'title':
      'automatic quality assessment of echocardiograms using convolutional neural networks: feasibility on the apical four-chamber view',
    'tools': ['caffe'],
    'year': 2017.0,
    'id': '3db1dca1-3ee6-416e-ac50-74359d00efb7.pdf',
    'use': 1.0,
    'domain': 'echocardiography',
    'task_1': 'quality assesment',
    'task_2': '0',
    'data_type': 'us',
    'location': 'heart',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '6,916',
    'training': 5533.0,
    'validation': 0.0,
    'testing': 1383.0,
    'cross validation': 1.0,
    'country_1': 'Canada',
    'country_2': '0',
    'country_3': 0.0,
    'journal': 'IEEE Transactions on Medical Imaging',
    'impact_factor': 3.942,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'The task of MRI fingerprinting is to identify tissue parameters from complex-valued MRI signals. The prevalent approach is dictionary based, where a test MRI signal is compared to stored MRI signals with known tissue parameters and the most similar signals and tissue parameters retrieved. Such an approach does not scale with the number of parameters and is rather slow when the tissue parameter space is large. Our first novel contribution is to use deep learning as an efficient nonlinear inverse mapping approach. We generate synthetic (tissue, MRI) data from an MRI simulator, and use them to train a deep net to map the MRI signal to the tissue parameters directly. Our second novel contribution is to develop a complex-valued neural network with new cardioid activation functions. Our results demonstrate that complex-valued neural nets could be much more accurate than real-valued neural nets at complex-valued MRI fingerprinting.',
    'authors': 'Patrick Virtue, Stella X. Yu, Michael Lustig',
    'description':
      'Accepted in Proc. IEEE International Conference on Image Processing (ICIP), 2017. 2017',
    'doi': null,
    'duplicate': false,
    'email': ['virtue@berkeley.edu'],
    'email_new': ['virtue@berkeley.edu'],
    'fullUrl': 'https://arxiv.org/abs/1707.00070',
    'key': '2b27e52f-7df6-4b60-9ed8-5900da98cb2c',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': 'https://arxiv.org/pdf/1707.00070.pdf',
    'phase': 1,
    'received': 0,
    'source': 'arxiv',
    'title':
      'better than real: complex-valued neural nets for mri fingerprinting',
    'tools': ['caffe'],
    'year': 2017.0,
    'id': '2b27e52f-7df6-4b60-9ed8-5900da98cb2c.pdf',
    'use': 1.0,
    'domain': 'radiology',
    'task_1': 'fingerprinting',
    'task_2': '0',
    'data_type': 'mri',
    'location': '0',
    'method_1': 'fullyconnectednn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 0.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '0',
    'training': 100000.0,
    'validation': 0.0,
    'testing': 0.0,
    'cross validation': 0.0,
    'country_1': 'USA',
    'country_2': '0',
    'country_3': 0.0,
    'journal': '0',
    'impact_factor': 0.0,
    'conference':
      ' IEEE International Conference on Image Processing (ICIP), 2017',
    'public': 1.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Automated breast cancer multi-classification from histopathological images plays a key role in computer-aided breast cancer diagnosis or prognosis. Breast cancer multi-classification is to identify subordinate classes of breast cancer (Ductal carcinoma, Fibroadenoma, Lobular carcinoma, etc.). However, breast cancer multi-classification from histopathological images faces two main challenges from: (1) the great difficulties in breast cancer multi-classification methods contrasting with the classification of binary classes (benign and malignant), and (2) the subtle differences in multiple classes due to the broad variability of high-resolution image appearances, high coherency of cancerous cells, and extensive inhomogeneity of color distribution. Therefore, automated breast cancer multi-classification from histopathological images is of great clinical significance yet has never been explored. Existing works in literature only focus on the binary classification but do not support further breast cancer quantitative assessment. In this study, we propose a breast cancer multi-classification method using a newly proposed deep learning model. The structured deep learning model has achieved remarkable performance (average 93.2% accuracy) on a large-scale dataset, which demonstrates the strength of our method in providing an efficient tool for breast cancer multi-classification in clinical settings.',
    'authors': 'Han Z, Wei B, Zheng Y, Yin Y, Li K, Li S.',
    'description': 'Sci Rep.  2017',
    'doi': null,
    'duplicate': false,
    'email': ['wbz99@sina.com'],
    'email_new': ['wbz99@sina.com'],
    'fullUrl': 'https://www.ncbi.nlm.nih.gov//pubmed/28646155',
    'key': '445e091e-d8a7-4e31-a501-26eaf91c882c',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'pubmed',
    'title':
      'breast cancer multi-classification from histopathological images with structured deep learning model',
    'tools': ['caffe'],
    'year': 2017.0,
    'id': '445e091e-d8a7-4e31-a501-26eaf91c882c.pdf',
    'use': 1.0,
    'domain': 'pathology',
    'task_1': 'classification',
    'task_2': '0',
    'data_type': 'histopathology',
    'location': 'breast',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 1.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1':
      'https://web.inf.ufpr.br/vri/databases/breast-cancer-histopathological-database-breakhis/',
    'data_link_2': '0',
    'data_ok': 1.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 1.0,
    'total': '82',
    'training': 61.0,
    'validation': 0.0,
    'testing': 21.0,
    'cross validation': 1.0,
    'country_1': 'China',
    'country_2': 'UK',
    'country_3': 0.0,
    'journal': 'nature scientific reports',
    'impact_factor': 4.259,
    'conference': '0',
    'public': 1.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Hyperspectral imaging is a rising imaging modality in the field of medical applications, and the combination of both spectral and spatial information provides wealth information for cell classification. In this paper, deep convolutional neural network (CNN) is employed to achieve blood cell discrimination in medical hyperspectral images (MHSI). As a deep learning architecture, CNNs are expected to get more discriminative and semantic features, which effect classification accuracy to a certain extent. Experimental results based on two real medical hyperspectral image data sets demonstrate that cell classification using CNNs is effective. In addition, compared to traditional support vector machine (SVM), the proposed method, which jointly exploits spatial and spectral features, can achieve better classification performance, showcasing the CNN-based methods\' tremendous potential for accurate medical hyperspectral data classification.',
    'authors': 'Xiang Li,  W. Li,  Xiaodong Xu,  Wei Hu',
    'description':
      '2017 2nd International Conference on Image, Vision and Computing (ICIVC). 2017',
    'doi': '10.1109/ICIVC.2017.7984606',
    'duplicate': false,
    'email': ['liwei089@ieee.org'],
    'email_new': ['liwei089@ieee.org'],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7984606',
    'key': '310fb8e5-7fda-416b-84c0-35cb33685984',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title':
      'cell classification using convolutional neural networks in medical hyperspectral imagery',
    'tools': ['caffe'],
    'year': 2017.0,
    'id': '310fb8e5-7fda-416b-84c0-35cb33685984.pdf',
    'use': 1.0,
    'domain': 'microscopy',
    'task_1': 'classification',
    'task_2': '0',
    'data_type': 'hsi',
    'location': 'blood',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '24222',
    'training': 9687.0,
    'validation': 0.0,
    'testing': 14535.0,
    'cross validation': 0.0,
    'country_1': 'China',
    'country_2': '0',
    'country_3': 0.0,
    'journal': '0',
    'impact_factor': 0.0,
    'conference':
      '2017 2nd International Conference on Image, Vision and Computing',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Dental records play an important role in forensic identification. To this end, postmortem dental findings and teeth conditions are recorded in a dental chart and compared with those of antemortem records. However, most dentists are inexperienced at recording the dental chart for corpses, and it is a physically and mentally laborious task, especially in large scale disasters. Our goal is to automate the dental filing process by using dental x-ray images. In this study, we investigated the application of a deep convolutional neural network (DCNN) for classifying tooth types on dental cone-beam computed tomography (CT) images. Regions of interest (ROIs) including single teeth were extracted from CT slices. Fifty two CT volumes were randomly divided into 42 training and 10 test cases, and the ROIs obtained from the training cases were used for training the DCNN. For examining the sampling effect, random sampling was performed 3 times, and training and testing were repeated. We used the AlexNet network architecture provided in the Caffe framework, which consists of 5 convolution layers, 3 pooling layers, and 2 full connection layers. For reducing the overtraining effect, we augmented the data by image rotation and intensity transformation. The test ROIs were classified into 7 tooth types by the trained network. The average classification accuracy using the augmented training data by image rotation and intensity transformation was 88.8%. Compared with the result without data augmentation, data augmentation resulted in an approximately 5% improvement in classification accuracy. This indicates that the further improvement can be expected by expanding the CT dataset. Unlike the conventional methods, the proposed method is advantageous in obtaining high classification accuracy without the need for precise tooth segmentation. The proposed tooth classification method can be useful in automatic filing of dental charts for forensic identification.Copyright \u00a9 2016 Elsevier Ltd. All rights reserved.',
    'authors':
      'Miki Y, Muramatsu C, Hayashi T, Zhou X, Hara T, Katsumata A, Fujita H.',
    'description': 'Comput Biol Med.  2017',
    'doi': null,
    'duplicate': false,
    'email': ['chisa@fjt.info.gifu-u.ac.jp'],
    'email_new': ['chisa@fjt.info.gifu-u.ac.jp'],
    'fullUrl': 'https://www.ncbi.nlm.nih.gov//pubmed/27889430',
    'key': '5d20cbe9-c3f0-41a5-9084-f623c01a9258',
    'keyok': 1,
    'keywords':
      'Deep convolutional neural networks; Dental chart; Dental cone-beam CT; Forensic identification; Tooth classification',
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'pubmed',
    'title':
      'classification of teeth in cone-beam ct using deep convolutional neural network',
    'tools': ['digits', 'caffe'],
    'year': 2017.0,
    'id': '5d20cbe9-c3f0-41a5-9084-f623c01a9258.pdf',
    'use': 1.0,
    'domain': 'dentistry',
    'task_1': 'classification',
    'task_2': '0',
    'data_type': 'ct',
    'location': 'teeth',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': 'https://github.com/BVLC/caffe/tree/master/models/bvlc_alexnet',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': null,
    'training': null,
    'validation': null,
    'testing': null,
    'cross validation': 1.0,
    'country_1': 'Japan',
    'country_2': '0',
    'country_3': 0.0,
    'journal': 'Computers in Biology and Medicine',
    'impact_factor': 1.836,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'INTRODUCTION: To develop real-time image processing for image-guided radiotherapy, we evaluated several neural network models for use with different imaging modalities, including X-ray fluoroscopic image denoising. MATERIALS: Setup images of prostate cancer patients were acquired with two oblique X-ray fluoroscopic units. Two types of residual network were designed: a convolutional autoencoder (rCAE) and a convolutional neural network (rCNN). We changed the convolutional kernel size and number of convolutional layers for both networks, and the number of pooling and upsampling layers for rCAE. The ground-truth image was applied to the contrast-limited adaptive histogram equalization (CLAHE) method of image processing. Network models were trained to keep the quality of the output image close to that of the ground-truth image from the input image without image processing. For image denoising evaluation, noisy input images were used for the training. RESULTS: More than 6 convolutional layers with convolutional kernels &gt;5\u00d75 improved image quality. However, this did not allow real-time imaging. After applying a pair of pooling and upsampling layers to both networks, rCAEs with &gt;3 convolutions each and rCNNs with &gt;12 convolutions with a pair of pooling and upsampling layers achieved real-time processing at 30 frames per second (fps) with acceptable image quality. CONCLUSIONS: Use of our suggested network achieved real-time image processing for contrast enhancement and image denoising by the use of a conventional modern personal computer. Copyright \u00a9 2017 Associazione Italiana di Fisica Medica. Published by Elsevier Ltd. All rights reserved.',
    'authors': 'Mori S.',
    'description': 'Phys Med.  2017',
    'doi': null,
    'duplicate': false,
    'email': ['mori.shinichiro@qst.go.jp'],
    'email_new': ['mori.shinichiro@qst.go.jp'],
    'fullUrl': 'https://www.ncbi.nlm.nih.gov//pubmed/28743618',
    'key': '3b168276-17e1-4dc7-91e9-d6dcb969b3e6',
    'keyok': 1,
    'keywords':
      'Computer-assisted; Fluoroscopy; Image processing; Neural network model; Radiation therapy',
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'pubmed',
    'title':
      'deep architecture neural network-based real-time image processing for image-guided radiotherapy',
    'tools': ['caffe'],
    'year': 2017.0,
    'id': '3b168276-17e1-4dc7-91e9-d6dcb969b3e6.pdf',
    'use': 1.0,
    'domain': 'image-guided radiotherapy',
    'task_1': 'denoising',
    'task_2': '0',
    'data_type': 'xray',
    'location': 'prostate',
    'method_1': 'rcae',
    'method_2': 'rcnn',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '42',
    'training': 0.0,
    'validation': 0.0,
    'testing': 0.0,
    'cross validation': 0.0,
    'country_1': 'Japan',
    'country_2': '0',
    'country_3': 0.0,
    'journal': 'Physica Medica',
    'impact_factor': 1.99,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'We propose a fully automatic method for segmenting myelin and axon from microscopy images of excised mouse spinal cord based on Convolutional Neural Networks (CNNs) and Deep Convolutional Encoder-Decoder. We compare a two-class CNN, multi-class CNN, and multi-class deep convolutional encoder-decoder with traditional methods. The CNN method gives a pixel-wise accuracy of 79.7% whereas an Active Contour method gives 59.4%. The encoder-decoder shows better performance with 82.3% and noticeably shorter classification time than CNN methods.',
    'authors': 'R. Mesbah,  B. McCane,  S. Mills',
    'description':
      '2016 International Conference on Image and Vision Computing New Zealand (IVCNZ). 2016',
    'doi': '10.1109/IVCNZ.2016.7804455',
    'duplicate': false,
    'email': [
      'rassoul@cs.otago.ac.nz',
      'mccane@cs.otago.ac.nz',
      'steven@cs.otago.ac.nz',
    ],
    'email_new': [
      'rassoul@cs.otago.ac.nz',
      'mccane@cs.otago.ac.nz',
      'steven@cs.otago.ac.nz',
    ],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7804455',
    'key': '04327d42-c350-4022-901a-6e46d79139bc',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title':
      'deep convolutional encoder-decoder for myelin and axon segmentation',
    'tools': ['torch'],
    'year': 2016.0,
    'id': '04327d42-c350-4022-901a-6e46d79139bc.pdf',
    'use': 1.0,
    'domain': 'microscopy',
    'task_1': 'segmentation',
    'task_2': '0',
    'data_type': 'microscopy',
    'location': 'mouse',
    'method_1': 'cnn',
    'method_2': 'dced',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '35',
    'training': 34.0,
    'validation': 0.0,
    'testing': 1.0,
    'cross validation': 1.0,
    'country_1': 'New Zealand',
    'country_2': '0',
    'country_3': 0.0,
    'journal': '0',
    'impact_factor': 0.0,
    'conference':
      '2016 International Conference on Image and Vision Computing New Zealand',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'PURPOSE: To describe and evaluate a new fully automated musculoskeletal tissue segmentation method using deep convolutional neural network (CNN) and three-dimensional (3D) simplex deformable modeling to improve the accuracy and efficiency of cartilage and bone segmentation within the knee joint. METHODS: A fully automated segmentation pipeline was built by combining a semantic segmentation CNN and 3D simplex deformable modeling. A CNN technique called SegNet was applied as the core of the segmentation method to perform high resolution pixel-wise multi-class tissue classification. The 3D simplex deformable modeling refined the output from SegNet to preserve the overall shape and maintain a desirable smooth surface for musculoskeletal structure. The fully automated segmentation method was tested using a publicly available knee image data set to compare with currently used state-of-the-art segmentation methods. The fully automated method was also evaluated on two different data sets, which include morphological and quantitative MR images with different tissue contrasts. RESULTS: The proposed fully automated segmentation method provided good segmentation performance with segmentation accuracy superior to most of state-of-the-art methods in the publicly available knee image data set. The method also demonstrated versatile segmentation performance on both morphological and quantitative musculoskeletal MR images with different tissue contrasts and spatial resolutions. CONCLUSION: The study demonstrates that the combined CNN and 3D deformable modeling approach is useful for performing rapid and accurate cartilage and bone segmentation within the knee joint. The CNN has promising potential applications in musculoskeletal imaging. Magn Reson Med, 2017. \u00a9 2017 International Society for Magnetic Resonance in Medicine.',
    'authors': 'Liu F, Zhou Z, Jang H, Samsonov A, Zhao G, Kijowski R.',
    'description': 'Magn Reson Med.  2017',
    'doi': null,
    'duplicate': false,
    'email': ['fliu37@wisc.edu'],
    'email_new': ['fliu37@wisc.edu'],
    'fullUrl': 'https://www.ncbi.nlm.nih.gov//pubmed/28733975',
    'key': '5adf95df-b744-476a-9738-34320b8a88e1',
    'keyok': 1,
    'keywords':
      'CNN; MRI; deep learning; deformable model; musculoskeletal imaging; segmentation',
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'pubmed',
    'title':
      'deep convolutional neural network and 3d deformable approach for tissue segmentation in musculoskeletal magnetic resonance imaging',
    'tools': ['caffe'],
    'year': 2017.0,
    'id': '5adf95df-b744-476a-9738-34320b8a88e1.pdf',
    'use': 1.0,
    'domain': 'radiology',
    'task_1': 'segmentation',
    'task_2': '0',
    'data_type': 'mri',
    'location': 'knee',
    'method_1': 'ced',
    'method_2': '0',
    'method_3': '0',
    'data_public': 1.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': 'www.ski10.org',
    'data_link_2': '0',
    'data_ok': 1.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': 'http://www.robots.ox.ac.uk/~vgg/research/very_deep/',
    'based_on_2': 'https://lmb.informatik.uni-freiburg.de/people/ronneber/u-net/',
    'transfer_learning': 0.0,
    'total': null,
    'training': null,
    'validation': null,
    'testing': null,
    'cross validation': 1.0,
    'country_1': 'USA',
    'country_2': '0',
    'country_3': 0.0,
    'journal': 'Magnetic Resonance in Medicine',
    'impact_factor': 3.924,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'The goal of this study is to evaluate the efficacy of deep convolutional neural networks (DCNNs) in differentiating subtle, intermediate, and more obvious image differences in radiography. Three different datasets were created, which included presence/absence of the endotracheal (ET) tube (n\u00a0=\u00a0300), low/normal position of the ET tube (n\u00a0=\u00a0300), and chest/abdominal radiographs (n\u00a0=\u00a0120). The datasets were split into training, validation, and test. Both untrained and pre-trained deep neural networks were employed, including AlexNet and GoogLeNet classifiers, using the Caffe framework. Data augmentation was performed for the presence/absence and low/normal ET tube datasets. Receiver operating characteristic (ROC), area under the curves (AUC), and 95% confidence intervals were calculated. Statistical differences of the AUCs were determined using a non-parametric approach. The pre-trained AlexNet and GoogLeNet classifiers had perfect accuracy (AUC 1.00) in differentiating chest vs. abdominal radiographs, using only 45 training cases. For more difficult datasets, including the presence/absence and low/normal position endotracheal tubes, more training cases, pre-trained networks, and data-augmentation approaches were helpful to increase accuracy. The best-performing network for classifying presence vs. absence of an ET tube was still very accurate with an AUC of 0.99. However, for the most difficult dataset, such as low vs. normal position of the endotracheal tube, DCNNs did not perform as well, but achieved a reasonable AUC of 0.81.',
    'authors': 'Lakhani P.',
    'description': 'J Digit Imaging.  2017',
    'doi': null,
    'duplicate': false,
    'email': ['Paras.lakhani@jefferson.edu'],
    'email_new': ['Paras.lakhani@jefferson.edu'],
    'fullUrl': 'https://www.ncbi.nlm.nih.gov//pubmed/28600640',
    'key': '4f1ffb01-099c-4b82-b32e-3f35295de368',
    'keyok': 1,
    'keywords':
      'Artificial intelligence; Artificial neural networks (ANNs); Classification; Machine learning; Radiography',
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'pubmed',
    'title':
      'deep convolutional neural networks for endotracheal tube position and x-ray image classification: challenges and opportunities',
    'tools': ['caffe'],
    'year': 2017.0,
    'id': '4f1ffb01-099c-4b82-b32e-3f35295de368.pdf',
    'use': 1.0,
    'domain': 'radiology',
    'task_1': 'classification',
    'task_2': '0',
    'data_type': 'xray',
    'location': 'chest',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': 'http://caffe.berkeleyvision.org/model_zoo.html',
    'based_on_2': '0',
    'transfer_learning': 1.0,
    'total': null,
    'training': null,
    'validation': null,
    'testing': null,
    'cross validation': 0.0,
    'country_1': 'USA',
    'country_2': '0',
    'country_3': 0.0,
    'journal': 'Journal of Digital Imaging',
    'impact_factor': 1.407,
    'conference': '0',
    'public': 1.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Automation-assisted cervical screening via Pap smear or liquid-based cytology (LBC) is a highly effective cell imaging based cancer detection tool, where cells are partitioned into \u201dabnormal\u201d and \u201dnormal\u201d categories. However, the success of most traditional classification methods relies on the presence of accurate cell segmentations. Despite sixty years of research in this field, accurate segmentation remains a challenge in the presence of cell clusters and pathologies. Moreover, previous classification methods are only built upon the extraction of hand-crafted features, such as morphology and texture. This paper addresses these limitations by proposing a method to directly classify cervical cells \u2013 without prior segmentation \u2013 based on deep features, using convolutional neural networks (ConvNets). First, the ConvNet is pre-trained on a natural image dataset. It is subsequently fine-tuned on a cervical cell dataset consisting of adaptively re-sampled image patches coarsely centered on the nuclei. In the testing phase, aggregation is used to average the prediction scores of a similar set of image patches. The proposed method is evaluated on both Pap smear and LBC datasets. Results show that our method outperforms previous algorithms in classification accuracy (98.3%), area under the curve (AUC) (0.99) values, and especially specificity (98.3%), when applied to the Herlev benchmark Pap smear dataset and evaluated using five-fold cross-validation. Similar superior performances are also achieved on the HEMLBC (H&E stained manual LBC) dataset. Our method is promising for the development of automation-assisted reading systems in primary cervical screening.',
    'authors': 'L. Zhang,  L. Lu,  I. Nogues,  R. Summers,  S. Liu,  J. Yao',
    'description': 'IEEE Journal of Biomedical and Health Informatics. 2017',
    'doi': '10.1109/JBHI.2017.2705583',
    'duplicate': false,
    'email': ['ling.zhang3@nih.gov', 'jyao@nih.gov'],
    'email_new': ['ling.zhang3@nih.gov', 'jyao@nih.gov'],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7932065',
    'key': '4b5193c9-e8bf-4b0a-9ce4-5cae475dd1b8',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title':
      'deeppap: deep convolutional networks for cervical cell classification',
    'tools': ['caffe', 'convnet'],
    'year': 2017.0,
    'id': '4b5193c9-e8bf-4b0a-9ce4-5cae475dd1b8.pdf',
    'use': 1.0,
    'domain': 'cytology',
    'task_1': 'classification',
    'task_2': '0',
    'data_type': 'pap smear',
    'location': 'cervix',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 1.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': 'http://mde-lab.aegean.gr/downloads',
    'data_link_2': '0',
    'data_ok': 1.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1':
      'https://github.com/BVLC/caffe/tree/master/models/bvlc_reference_caffenet',
    'based_on_2': '0',
    'transfer_learning': 1.0,
    'total': null,
    'training': null,
    'validation': null,
    'testing': null,
    'cross validation': 1.0,
    'country_1': 'USA',
    'country_2': 'China',
    'country_3': 0.0,
    'journal': 'IEEE Journal of Biomedical and Health Informatics',
    'impact_factor': 3.451,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'The early diagnosis of Alzheimer\'s Disease (AD) and its prodromal form, Mild Cognitive Impairment (MCI), has been the subject of extensive research in recent years. Some recent studies have shown promising results in the diagnosis of AD and MCI using structural Magnetic Resonance Imaging (MRI) scans. In this paper, we propose the use of a Convolutional Neural Network (CNN) in the detection of AD and MCI. In particular, we modified the 16-layered VGGNet for the 3-way classification of AD, MCI and Healthy Controls (HC) on the Alzheimer\'s Disease Neuroimaging Initiative (ADNI) dataset achieving an overall accuracy of 91.85% and outperforming several classifiers from other studies.',
    'authors':
      'C. D. Billones,  O. J. L. D. Demetria,  D. E. D. Hostallero,  P. C. Naval',
    'description': '2016 IEEE Region 10 Conference (TENCON). 2016',
    'doi': '10.1109/TENCON.2016.7848755',
    'duplicate': false,
    'email': ['pcnaval@dcs.upd.edu.ph'],
    'email_new': ['pcnaval@dcs.upd.edu.ph'],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7848755',
    'key': '279c6394-f97b-4307-99d8-71d593434033',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title':
      'demnet: a convolutional neural network for the detection of alzheimer\'s disease and mild cognitive impairment',
    'tools': ['digits', 'caffe'],
    'year': 2016.0,
    'id': '279c6394-f97b-4307-99d8-71d593434033.pdf',
    'use': 1.0,
    'domain': 'radiology',
    'task_1': 'classification',
    'task_2': '0',
    'data_type': 'mri',
    'location': 'brain',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 1.0,
    'data_shared': 0.0,
    'data_link_1': 'adni.loni.usc.edu',
    'data_link_2': '0',
    'data_ok': 1.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': 'http://www.robots.ox.ac.uk/~vgg/research/very_deep/',
    'based_on_2': '0',
    'transfer_learning': 1.0,
    'total': '900',
    'training': 630.0,
    'validation': 0.0,
    'testing': 270.0,
    'cross validation': 0.0,
    'country_1': 'Philippines',
    'country_2': '0',
    'country_3': 0.0,
    'journal': '0',
    'impact_factor': 0.0,
    'conference': '2016 IEEE Region 10 Conference (TENCON)',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'PURPOSE: Colitis refers to inflammation of the inner lining of the colon that is frequently associated with infection and allergic reactions. In this paper, we propose deep convolutional neural networks methods for lesion-level colitis detection and a support vector machine (SVM) classifier for patient-level colitis diagnosis on routine abdominal CT scans. METHODS: The recently developed Faster Region-based Convolutional Neural Network (Faster RCNN) is utilized for lesion-level colitis detection. For each 2D slice, rectangular region proposals are generated by region proposal networks (RPN). Then, each region proposal is jointly classified and refined by a softmax classifier and bounding-box regressor. Two convolutional neural networks, eight layers of ZF net and 16 layers of VGG net are compared for colitis detection. Finally, for each patient, the detections on all 2D slices are collected and a SVM classifier is applied to develop a patient-level diagnosis. We trained and evaluated our method with 80 colitis patients and 80 normal cases using 4\u00a0\u00d7\u00a04-fold cross validation. RESULTS: For lesion-level colitis detection, with ZF net, the mean of average precisions (mAP) were 48.7% and 50.9% for RCNN and Faster RCNN, respectively. The detection system achieved sensitivities of 51.4% and 54.0% at two false positives per patient for RCNN and Faster RCNN, respectively. With VGG net, Faster RCNN increased the mAP to 56.9% and increased the sensitivity to 58.4% at two false positive per patient. For patient-level colitis diagnosis, with ZF net, the average areas under the ROC curve (AUC) were 0.978\u00a0\u00b1\u00a00.009 and 0.984\u00a0\u00b1\u00a00.008 for RCNN and Faster RCNN method, respectively. The difference was not statistically significant with P\u00a0=\u00a00.18. At the optimal operating point, the RCNN method correctly identified 90.4% (72.3/80) of the colitis patients and 94.0% (75.2/80) of normal cases. The sensitivity improved to 91.6% (73.3/80) and the specificity improved to 95.0% (76.0/80) for the Faster RCNN method. With VGG net, Faster RCNN increased the AUC to 0.986\u00a0\u00b1\u00a00.007 and increased the diagnosis sensitivity to 93.7% (75.0/80) and specificity was unchanged at 95.0% (76.0/80). CONCLUSION: Colitis detection and diagnosis by deep convolutional neural networks is accurate and promising for future clinical application. Published 2017. This article is a U.S. Government work and is in the public domain in the USA.',
    'authors':
      'Liu J, Wang D, Lu L, Wei Z, Kim L, Turkbey EB, Sahiner B, Petrick NA, Summers RM.',
    'description': 'Med Phys.  2017',
    'doi': null,
    'duplicate': false,
    'email': ['rms@nih.gov'],
    'email_new': ['rms@nih.gov'],
    'fullUrl': 'https://www.ncbi.nlm.nih.gov//pubmed/28594460',
    'key': '53aaa619-cfc3-4fa7-93ad-a399933683d3',
    'keyok': 1,
    'keywords': 'RPN; Region-based CNN; colitis detection; colitis diagnosis',
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'pubmed',
    'title':
      'detection and diagnosis of colitis on computed tomography using deep convolutional neural networks',
    'tools': ['caffe'],
    'year': 2017.0,
    'id': '53aaa619-cfc3-4fa7-93ad-a399933683d3.pdf',
    'use': 1.0,
    'domain': 'radiology',
    'task_1': 'detection',
    'task_2': 'diagnosis',
    'data_type': 'ct',
    'location': 'colon',
    'method_1': 'regioncnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': 'http://www.robots.ox.ac.uk/~vgg/research/very_deep/',
    'based_on_2': '0',
    'transfer_learning': 1.0,
    'total': null,
    'training': null,
    'validation': null,
    'testing': null,
    'cross validation': 1.0,
    'country_1': 'USA',
    'country_2': '0',
    'country_3': 0.0,
    'journal': 'medical physics ',
    'impact_factor': 2.635,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'BACKGROUND AND OBJECTIVE: Diabetic retinopathy is one of the leading disabling chronic diseases and one of the leading causes of preventable blindness in developed world. Early diagnosis of diabetic retinopathy enables timely treatment and in order to achieve it a major effort will have to be invested into automated population screening programs. Detection of exudates in color fundus photographs is very important for early diagnosis of diabetic retinopathy. METHODS: We use deep convolutional neural networks for exudate detection. In order to incorporate high level anatomical knowledge about potential exudate locations, output of the convolutional neural network is combined with the output of the optic disc detection and vessel detection procedures. RESULTS: In the validation step using a manually segmented image database we obtain a maximum F1 measure of 0.78. CONCLUSIONS: As manually segmenting and counting exudate areas is a tedious task, having a reliable automated output, such as automated segmentation using convolutional neural networks in combination with other landmark detectors, is an important step in creating automated screening programs for early detection of diabetic retinopathy. Copyright \u00a9 2016 Elsevier Ireland Ltd. All rights reserved.',
    'authors': 'Prenta\u0161i\u0107 P, Lon\u010dari\u0107 S.',
    'description': 'Comput Methods Programs Biomed.  2016',
    'doi': null,
    'duplicate': false,
    'email': ['pavle.prentasic@fer.hr'],
    'email_new': ['pavle.prentasic@fer.hr'],
    'fullUrl': 'https://www.ncbi.nlm.nih.gov//pubmed/28110732',
    'key': '15de50b5-be30-42fc-b6e0-ab0f0770d7d0',
    'keyok': 1,
    'keywords':
      'Convolutional neural networks; Diabetic retinopathy; Exudates; Fundus photographs; Machine learning',
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'pubmed',
    'title':
      'detection of exudates in fundus photographs using deep neural networks and anatomical landmark detection fusion',
    'tools': ['caffe'],
    'year': 2016.0,
    'id': '15de50b5-be30-42fc-b6e0-ab0f0770d7d0.pdf',
    'use': 1.0,
    'domain': 'ophthalmology',
    'task_1': 'detection',
    'task_2': '0',
    'data_type': 'fundus photography',
    'location': 'eye',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 1.0,
    'data_shared': 0.0,
    'data_link_1': 'http://ieeexplore.ieee.org/document/6703830/',
    'data_link_2': '0',
    'data_ok': 1.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '50',
    'training': 0.0,
    'validation': 0.0,
    'testing': 0.0,
    'cross validation': 1.0,
    'country_1': 'Croatia',
    'country_2': '0',
    'country_3': 0.0,
    'journal': 'Computer Methods and Programs in Biomedicine',
    'impact_factor': 2.503,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'BACKGROUND: Ongoing research into inflammatory conditions raises an increasing need to evaluate immune cells in histological sections in biologically relevant regions of interest (ROIs). Herein, we compare different approaches to automatically detect lobular structures in human normal breast tissue in digitized whole slide images (WSIs). This automation is required to perform objective and consistent quantitative studies on large data sets. METHODS: In normal breast tissue from nine healthy patients immunohistochemically stained for different markers, we evaluated and compared three different image analysis methods to automatically detect lobular structures in WSIs: (1) a bottom-up approach using the cell-based data for subsequent tissue level classification, (2) a top-down method starting with texture classification at tissue level analysis of cell densities in specific ROIs, and (3) a direct texture classification using deep learning technology. RESULTS: All three methods result in comparable overall quality allowing automated detection of lobular structures with minor advantage in sensitivity (approach 3), specificity (approach 2), or processing time (approach 1). Combining the outputs of the approaches further improved the precision. CONCLUSIONS: Different approaches of automated ROI detection are feasible and should be selected according to the individual needs of biomarker research. Additionally, detected ROIs could be used as a basis for quantification of immune infiltration in lobular structures. Copyright \u00a9 2016 Elsevier Ltd. All rights reserved.',
    'authors':
      'Apou G, Schaadt NS, Naegel B, Forestier G, Sch\u00f6nmeyer R, Feuerhake F, Wemmert C, Grote A.',
    'description': 'Comput Biol Med.  2016',
    'doi': null,
    'duplicate': false,
    'email': ['gapou@unistra.fr'],
    'email_new': ['gapou@unistra.fr'],
    'fullUrl': 'https://www.ncbi.nlm.nih.gov//pubmed/27209271',
    'key': '48752b7a-3043-4a6f-ae1c-49f1d134b915',
    'keyok': 1,
    'keywords':
      'Convolutional neural network; Digital histopathology; Image analysis; Normal breast lobule; Whole slide image',
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'pubmed',
    'title': 'detection of lobular structures in normal breast tissue',
    'tools': ['caffe'],
    'year': 2016.0,
    'id': '48752b7a-3043-4a6f-ae1c-49f1d134b915.pdf',
    'use': 1.0,
    'domain': 'pathology',
    'task_1': 'detection',
    'task_2': '0',
    'data_type': 'wsi',
    'location': 'breast',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '9',
    'training': 0.0,
    'validation': 0.0,
    'testing': 0.0,
    'cross validation': 1.0,
    'country_1': 'Germany',
    'country_2': 'France',
    'country_3': 0.0,
    'journal': 'Computers in Biology and Medicine',
    'impact_factor': 1.836,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Epithelium-stroma classification is a necessary preprocessing step in histopathological image analysis. Current deep learning based recognition methods for histology data require collection of large volumes of labeled data in order to train a new neural network when there are changes to the image acquisition procedure. However, it is extremely expensive for pathologists to manually label sufficient volumes of data for each pathology study in a professional manner, which results in limitations in real-world applications. A very simple but effective deep learning method, that introduces the concept of unsupervised domain adaptation to a simple convolutional neural network (CNN), has been proposed in this paper. Inspired by transfer learning, our work assumes that the training data and testing data follow different distributions, and there is an adaptation operation to more accurately estimate the kernels in CNN in feature extraction, in order to enhance performance by transferring knowledge from labeled data in source domain to unlabeled data in target domain. The model has been evaluated using three independent public epithelium-stroma datasets by cross-dataset validations. The experimental results demonstrate that for epithelium-stroma classification, the proposed framework outperforms the state-of-the-art deep neural network model, and it also achieves better performance than other existing deep domain adaptation methods. The proposed model can be considered to be a better option for real-world applications in histopathological image analysis, since there is no longer a requirement for large-scale labeled data in each specified domain.',
    'authors': 'Huang Y, Zheng H, Liu C, Ding X, Rohde G.',
    'description': 'IEEE J Biomed Health Inform.  2017',
    'doi': null,
    'duplicate': false,
    'email': ['yhuang2010@xmu.edu.cn', 'dxh@xmu.edu.cn'],
    'email_new': ['yhuang2010@xmu.edu.cn', 'dxh@xmu.edu.cn'],
    'fullUrl': 'https://www.ncbi.nlm.nih.gov//pubmed/28410112',
    'key': '361c7ffa-8455-4deb-a7cf-37fc6604adcb',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'pubmed',
    'title':
      'epithelium-stroma classification via convolutional neural networks and unsupervised domain adaptation in histopathological images',
    'tools': ['caffe'],
    'year': 2017.0,
    'id': '361c7ffa-8455-4deb-a7cf-37fc6604adcb.pdf',
    'use': 1.0,
    'domain': 'pathology',
    'task_1': 'classification',
    'task_2': '0',
    'data_type': 'histopathology',
    'location': 'cell',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 1.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': 'https://github.com/KaimingHe/deep-residual-networks',
    'based_on_2': '0',
    'transfer_learning': 1.0,
    'total': '1219',
    'training': 0.0,
    'validation': 0.0,
    'testing': 0.0,
    'cross validation': 1.0,
    'country_1': 'China',
    'country_2': 'USA',
    'country_3': 0.0,
    'journal': 'IEEE Journal of Biomedical and Health Informatics',
    'impact_factor': 3.451,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'The interpretation and analysis of wireless capsule endoscopy (WCE) recordings is a complex task which requires sophisticated computer aided decision (CAD) systems to help physicians with video screening and, finally, with the diagnosis. Most CAD systems used in capsule endoscopy share a common system design, but use very different image and video representations. As a result, each time a new clinical application of WCE appears, a new CAD system has to be designed from the scratch. This makes the design of new CAD systems very time consuming. Therefore, in this paper we introduce a system for small intestine motility characterization, based on Deep Convolutional Neural Networks, which circumvents the laborious step of designing specific features for individual motility events. Experimental results show the superiority of the learned features over alternative classifiers constructed using state-of-the-art handcrafted features. In particular, it reaches a mean classification accuracy of 96% for six intestinal motility events, outperforming the other classifiers by a large margin (a 14% relative performance increase). Copyright \u00a9 2016 Elsevier Ltd. All rights reserved.',
    'authors':
      'Segu\u00ed S, Drozdzal M, Pascual G, Radeva P, Malagelada C, Azpiroz F, Vitri\u00e0 J.',
    'description': 'Comput Biol Med.  2016',
    'doi': null,
    'duplicate': false,
    'email': ['santi.segui@ub.edu'],
    'email_new': ['santi.segui@ub.edu'],
    'fullUrl': 'https://www.ncbi.nlm.nih.gov//pubmed/27810622',
    'key': '00de577b-26a3-47ce-9a98-27259e3c3c3b',
    'keyok': 1,
    'keywords':
      'Deep learning; Feature learning; Motility analysis; Wireless capsule endoscopy',
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'pubmed',
    'title': 'generic feature learning for wireless capsule endoscopy analysis',
    'tools': ['caffe'],
    'year': 2016.0,
    'id': '00de577b-26a3-47ce-9a98-27259e3c3c3b.pdf',
    'use': 1.0,
    'domain': 'endoscopy',
    'task_1': 'detection',
    'task_2': '0',
    'data_type': 'wireless capsule endoscopy',
    'location': 'gastrointestinal',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '120000',
    'training': 100000.0,
    'validation': 0.0,
    'testing': 20000.0,
    'cross validation': 0.0,
    'country_1': 'Spain',
    'country_2': 'Israel',
    'country_3': 0.0,
    'journal': 'Computers in Biology and Medicine',
    'impact_factor': 1.836,
    'conference': '0',
    'public': 1.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Given the potential X-ray radiation risk to the patient, low-dose CT has attracted a considerable interest in the medical imaging field. The current main stream low-dose CT methods include vendor-specific sinogram domain filtration and iterative reconstruction, but they need to access original raw data whose formats are not transparent to most users. Due to the difficulty of modeling the statistical characteristics in the image domain, the existing methods for directly processing reconstructed images cannot eliminate image noise very well while keeping structural details. Inspired by the idea of deep learning, here we combine the autoencoder, the deconvolution network, and shortcut connections into the residual encoder-decoder convolutional neural network (RED-CNN) for low-dose CT imaging. After patch-based training, the proposed RED-CNN achieves a competitive performance relative to the-state-of-art methods in both simulated and clinical cases. Especially, our method has been favorably evaluated in terms of noise suppression, structural preservation and lesion detection.',
    'authors':
      'Hu Chen, Yi Zhang, Mannudeep K. Kalra, Feng Lin, Yang Chen, Peixi Liao, Jiliu Zhou, Ge Wang',
    'description': 'Accepted by IEEE TMI. 2017',
    'doi': null,
    'duplicate': false,
    'email': [
      'linfeng@scu.edu.cn',
      'huchen@scu.edu.cn',
      'zhoujl@scu.edu.cn',
      'yzhang@scu.edu.cn',
      'mkalra@mgh.harvard.edu',
      'chenyang.list@seu.edu.cn',
      'universe6527@163.com',
      'wangg6@rpi.edu',
    ],
    'email_new': [
      'linfeng@scu.edu.cn',
      'huchen@scu.edu.cn',
      'zhoujl@scu.edu.cn',
      'yzhang@scu.edu.cn',
      'mkalra@mgh.harvard.edu',
      'chenyang.list@seu.edu.cn',
      'universe6527@163.com',
      'wangg6@rpi.edu',
    ],
    'fullUrl': 'https://arxiv.org/abs/1702.00288',
    'key': '35f064f0-3981-4161-89c8-d1be8dff46bd',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': 'https://arxiv.org/pdf/1702.00288.pdf',
    'phase': 1,
    'received': 0,
    'source': 'arxiv',
    'title':
      'low-dose ct with a residual encoder-decoder convolutional neural network (red-cnn)',
    'tools': ['caffe'],
    'year': 2017.0,
    'id': '35f064f0-3981-4161-89c8-d1be8dff46bd.pdf',
    'use': 1.0,
    'domain': 'radiology',
    'task_1': 'denoising',
    'task_2': '0',
    'data_type': 'ct',
    'location': '0',
    'method_1': 'redcnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 1.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': 'https://imaging.nci.nih.gov/ncia/login.jsf',
    'data_link_2': '0',
    'data_ok': 1.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '175',
    'training': 0.0,
    'validation': 0.0,
    'testing': 0.0,
    'cross validation': 1.0,
    'country_1': 'China',
    'country_2': 'USA',
    'country_3': 0.0,
    'journal': 'IEEE Transactions on Medical Imaging',
    'impact_factor': 3.942,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'With the development of various imaging technologies, medical imaging has been playing more important roles on providing scientific proof for doctors to make decisions on clinical diagnosis. At the same time, it is very important to excavate valuable information hidden in those images and take over some auxiliary medical works from doctors by the computer. Therefore, a large number of image segmentation methods, including some classic algorithms have been proposed and some of them perform well. In this paper, we built a deep convolutional neural network to segment the MRI brain images. Results show that the network has a good performance on segmentation of the gray and white matter of brains, it also had a good generalization ability.',
    'authors': 'Y. Wang,  Z. Sun,  C. Liu,  W. Peng,  J. Zhang',
    'description':
      '2016 IEEE International Conference on Mechatronics and Automation. 2016',
    'doi': '10.1109/ICMA.2016.7558819',
    'duplicate': false,
    'email': ['jhzhang@bit.edu.cn'],
    'email_new': ['jhzhang@bit.edu.cn'],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7558819',
    'key': '2d681b43-b383-48fc-9833-1b45c9eaf8e9',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title': 'mri image segmentation by fully convolutional networks',
    'tools': ['caffe'],
    'year': 2016.0,
    'id': '2d681b43-b383-48fc-9833-1b45c9eaf8e9.pdf',
    'use': 1.0,
    'domain': 'radiology',
    'task_1': 'segmentation',
    'task_2': '0',
    'data_type': 'mri',
    'location': 'brain',
    'method_1': 'fullycnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 1.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': 'https://www.nitrc.org/',
    'data_link_2': '0',
    'data_ok': 1.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '18',
    'training': 4.0,
    'validation': 0.0,
    'testing': 14.0,
    'cross validation': 0.0,
    'country_1': 'China',
    'country_2': '0',
    'country_3': 0.0,
    'journal': '0',
    'impact_factor': 0.0,
    'conference':
      '2016 IEEE International Conference on Mechatronics and Automation',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Recent advances in machine learning, specifically in deep learning with neural networks, has made a profound impact on fields such as natural language processing, image classification, and language modeling; however, feasibility and potential benefits of the approaches to metagenomic data analysis has been largely under-explored. Deep learning exploits many layers of learning nonlinear feature representations, typically in an unsupervised fashion, and recent results have shown outstanding generalization performance on previously unseen data. Furthermore, some deep learning methods can also represent the structure in a data set. Consequently, deep learning and neural networks may prove to be an appropriate approach for metagenomic data. To determine whether such approaches are indeed appropriate for metagenomics, we experiment with two deep learning methods: i) a deep belief network, and ii) a recursive neural network, the latter of which provides a tree representing the structure of the data. We compare these approaches to the standard multi-layer perceptron, which has been well-established in the machine learning community as a powerful prediction algorithm, though its presence is largely missing in metagenomics literature. We find that traditional neural networks can be quite powerful classifiers on metagenomic data compared to baseline methods, such as random forests. On the other hand, while the deep learning approaches did not result in improvements to the classification accuracy, they do provide the ability to learn hierarchical representations of a data set that standard classification methods do not allow. Our goal in this effort is not to determine the best algorithm in terms accuracy-as that depends on the specific application-but rather to highlight the benefits and drawbacks of each of the approach we discuss and provide insight on how they can be improved for predictive metagenomic analysis.',
    'authors': 'G. Ditzler,  R. Polikar,  G. Rosen',
    'description': 'IEEE Transactions on NanoBioscience. 2015',
    'doi': '10.1109/TNB.2015.2461219',
    'duplicate': false,
    'email': [
      'gregory.ditzler@gmail.com',
      'gailr@ece.drexel.edu',
      'polikar@rowan.edu',
    ],
    'email_new': [
      'gregory.ditzler@gmail.com',
      'gailr@ece.drexel.edu',
      'polikar@rowan.edu',
    ],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7219432',
    'key': '454721f9-3a79-4bdf-9e83-db9f2996e93c',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title':
      'multi-layer and recursive neural networks for metagenomic classification',
    'tools': ['torch'],
    'year': 2015.0,
    'id': '454721f9-3a79-4bdf-9e83-db9f2996e93c.pdf',
    'use': 1.0,
    'domain': 'metagenomics',
    'task_1': 'classification',
    'task_2': '0',
    'data_type': 'dna',
    'location': 'microorganisms',
    'method_1': 'dbn',
    'method_2': 'rnn',
    'method_3': 'mlp',
    'data_public': 1.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': 'https://qiita.ucsd.edu/',
    'data_link_2': '0',
    'data_ok': 1.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': 'https://code.google.com/archive/p/matrbm/',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '0',
    'training': 0.0,
    'validation': 0.0,
    'testing': 0.0,
    'cross validation': 1.0,
    'country_1': 'USA',
    'country_2': '0',
    'country_3': 0.0,
    'journal': 'IEEE Transactions on NanoBioscience',
    'impact_factor': 2.771,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Automatic diabetes retinopathy (DR) recognition can help DR carriers to receive treatment in early stages and avoid the risk of vision loss. In this paper, we emphasize the role of multiple filter sizes in learning fine-grained discriminant features and propose: (i) two deep convolutional neural networks - Combined Kernels with Multiple Losses Network (CKML Net) and VGGNet with Extra Kernel (VNXK), which are an improvement upon GoogLeNet and VGGNet in context of DR tasks. Learning from existing research, (ii) we propose a hybrid color space, LGI, for DR recognition via proposed nets. (iii) Transfer learning is applied to solve the challenge of imbalanced dataset. The effectiveness of proposed new nets and color space is evaluated using two grand challenge retina datasets: EyePACS and Messidor. Our experimental results show: (iv) CKML Net improves upon GoogLeNet and VNXK improves upon VGGNet on both datasets using the LGI color space. Additionally, proposed methodology improves upon other state of the art results on Messidor dataset for referable/non-referable screening.',
    'authors': 'H. H. Vo,  A. Verma',
    'description': '2016 IEEE International Symposium on Multimedia (ISM). 2016',
    'doi': '10.1109/ISM.2016.0049',
    'duplicate': false,
    'email': ['hhvo@csu.fullerton.edu', 'averma@fullerton.edu'],
    'email_new': ['hhvo@csu.fullerton.edu', 'averma@fullerton.edu'],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7823616',
    'key': '61e88b6c-394f-41be-888d-82cd96c0c4b1',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title':
      'new deep neural nets for fine-grained diabetic retinopathy recognition on hybrid color space',
    'tools': ['digits', 'caffe'],
    'year': 2016.0,
    'id': '61e88b6c-394f-41be-888d-82cd96c0c4b1.pdf',
    'use': 1.0,
    'domain': 'ophthalmology',
    'task_1': 'detection',
    'task_2': '0',
    'data_type': 'fundus photography',
    'location': 'eye',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 1.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': 'https://www.kaggle.com/c/diabetic-retinopathy-detection',
    'data_link_2':
      'http://www.adcis.net/en/Download-Third-Party/Messidor.htmlindex-en.php',
    'data_ok': 1.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': 'http://www.robots.ox.ac.uk/~vgg/research/very_deep/',
    'based_on_2': '0',
    'transfer_learning': 1.0,
    'total': null,
    'training': null,
    'validation': null,
    'testing': null,
    'cross validation': 0.0,
    'country_1': 'USA',
    'country_2': '0',
    'country_3': 0.0,
    'journal': '0',
    'impact_factor': 0.0,
    'conference': '2016 IEEE International Symposium on Multimedia',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'As a part of the automatic study of visual attention of affected populations with neurodegenerative diseases and to predict whether new gaze records a complaint of these diseases, we should design an automatic model that predicts salient areas in video. Past research showed, that people suffering form dementia are not reactive with regard to degradations on still images. In this paper we study the reaction of healthy normal control subjects on degraded area in videos. Furthermore, in the goal to build an automatic prediction model for salient areas in intentionally degraded videos, we design a deep learning architecture and measure its performances when predicting salient regions on completely unseen data. The obtained results are interesting regarding the reaction of normal control subjects against a degraded area in video.',
    'authors': 'S. Chaabouni,  F. Tison,  J. Benois-Pineau,  C. Ben Amar',
    'description':
      '2016 14th International Workshop on Content-Based Multimedia Indexing (CBMI). 2016',
    'doi': '10.1109/CBMI.2016.7500243',
    'duplicate': false,
    'email': [
      'souad.chaabouni@labri.fr',
      'benois-p@labri.fr',
      'francois.tison@chu-bordeaux.fr',
      'chokri.benamar@ieee.org',
    ],
    'email_new': [
      'souad.chaabouni@labri.fr',
      'benois-p@labri.fr',
      'francois.tison@chu-bordeaux.fr',
      'chokri.benamar@ieee.org',
    ],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7500243',
    'key': '21ca4519-4415-4c97-8d05-a26b35ebefae',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title':
      'prediction of visual attention with deep cnn for studies of neurodegenerative diseases',
    'tools': ['caffe'],
    'year': 2016.0,
    'id': '21ca4519-4415-4c97-8d05-a26b35ebefae.pdf',
    'use': 1.0,
    'domain': 'neurology',
    'task_1': 'saliency',
    'task_2': '0',
    'data_type': 'video',
    'location': 'gaze',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 1.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': 'http://www.di.ens.fr/~laptev/actions/hollywood2/',
    'data_link_2': '0',
    'data_ok': 1.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 0.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '1707',
    'training': 823.0,
    'validation': 0.0,
    'testing': 884.0,
    'cross validation': 0.0,
    'country_1': 'France',
    'country_2': 'Tunisia',
    'country_3': 0.0,
    'journal': '0',
    'impact_factor': 0.0,
    'conference':
      '2016 14th International Workshop on Content-Based Multimedia Indexing (CBMI)',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'R-CNN (region-convolutional neural network) has recently achieved very outstanding results in variety of visual detecting fields, and its function of object-proposal-generation can achieve effective training models by using as small samples as possible in the field of machine learning. In this paper, a modified R-CNN is proposed and applied to detect cells under phase contrast microscopy images by adopting multiple object-proposal-generations instead of a single one to extract candidate regions. The results show that the proposed method can obtain better performance than the traditional method by using a single object-proposal-generation.',
    'authors': 'F. Deng,  H. Hu,  S. Chen,  Q. Guan,  Y. Zou',
    'description':
      '2015 Sixth International Conference on Intelligent Control and Information Processing (ICICIP). 2015',
    'doi': '10.1109/ICICIP.2015.7388195',
    'duplicate': false,
    'email': ['csy@zjut.edu.cn'],
    'email_new': ['csy@zjut.edu.cn'],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7388195',
    'key': '2a2d0c01-4d6f-4ee5-9ecd-7652866e82ac',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title':
      'rich feature hierarchies for cell detecting under phase contrast microscopy images',
    'tools': ['caffe'],
    'year': 2015.0,
    'id': '2a2d0c01-4d6f-4ee5-9ecd-7652866e82ac.pdf',
    'use': 1.0,
    'domain': 'microscopy',
    'task_1': 'detection',
    'task_2': '0',
    'data_type': 'phase contrast microscopy',
    'location': '0',
    'method_1': 'regioncnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '100',
    'training': 80.0,
    'validation': 0.0,
    'testing': 20.0,
    'cross validation': 0.0,
    'country_1': 'China',
    'country_2': '0',
    'country_3': 0.0,
    'journal': '0',
    'impact_factor': 0.0,
    'conference':
      '2015 Sixth International Conference on Intelligent Control and Information Processing (ICICIP)',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Vessel segmentation of digital retinal images plays an important role in diagnosis of diseases such as diabetics, hypertension and retinopathy of prematurity due to these diseases impact the retina. In this paper, a novel Size-Invariant Fully Convolutional Neural Network (SIFCN) is proposed to address the automatic retinal vessel segmentation problems. The input data of the network is the patches of images and the corresponding pixel-wise labels. A consecutive convolution layers and pooling layers follow the input data, so that the network can learn the abstract features to segment retinal vessel. Our network is designed to hold the height and width of data of each layer with padding and assign pooling stride so that the spatial information maintain and up-sample is not required. Compared with the pixel-wise retinal vessel segmentation approaches, our patch-wise segmentation is much more efficient since in each cycle it can predict all the pixels of the patch. Our overlapped SIFCN approach achieves accuracy of 0.9471, with the AUC of 0.9682. And our non-overlap SIFCN is the most efficient approach among the deep learning approaches, costing only 3.68 seconds per image, and the overlapped SIFCN costs 31.17 seconds per image.',
    'authors': 'Y. Luo,  H. Cheng,  L. Yang',
    'description':
      '2016 Asia-Pacific Signal and Information Processing Association Annual Summit and Conference (APSIPA). 2016',
    'doi': '10.1109/APSIPA.2016.7820677',
    'duplicate': false,
    'email': ['1099361041@qq.com', 'hcheng@uestc.edu.cn', 'yanglu@uestc.edu.cn'],
    'email_new': [
      '1099361041@qq.com',
      'hcheng@uestc.edu.cn',
      'yanglu@uestc.edu.cn',
    ],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7820677',
    'key': '4c314e2b-7f3b-437e-bf24-b341ea4dbe84',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title':
      'size-invariant fully convolutional neural network for vessel segmentation of digital retinal images',
    'tools': ['caffe'],
    'year': 2016.0,
    'id': '4c314e2b-7f3b-437e-bf24-b341ea4dbe84.pdf',
    'use': 1.0,
    'domain': 'ophthalmology',
    'task_1': 'segmentation',
    'task_2': '0',
    'data_type': 'fundus photography',
    'location': 'eye',
    'method_1': 'fullycnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 1.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': 'http://www.isi.uu.nl/Research/Databases/DRIVE/',
    'data_link_2': '0',
    'data_ok': 1.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': null,
    'training': null,
    'validation': null,
    'testing': null,
    'cross validation': 0.0,
    'country_1': 'China',
    'country_2': '0',
    'country_3': 0.0,
    'journal': '0',
    'impact_factor': 0.0,
    'conference':
      '2016 Asia-Pacific Signal and Information Processing Association Annual Summit and Conference (APSIPA)',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'A baby\'s gestational age determines whether or not they are preterm, which helps clinicians decide on suitable post-natal treatment. The most accurate dating methods use Ultrasound Scan (USS) machines, but these machines are expensive, require trained personnel and cannot always be deployed to remote areas. In the absence of USS, the Ballard Score can be used, which is a manual postnatal dating method. However, this method is highly subjective and results can vary widely depending on the experience of the rater. In this paper, we present an automatic system for postnatal gestational age estimation aimed to be deployed on mobile phones, using small sets of images of a newborn\'s face, foot and ear. We present a novel two-stage approach that makes the most out of Convolutional Neural Networks trained on small sets of images to predict broad classes of gestational age, and then fuse the outputs of these discrete classes with a baby\'s weight to make fine-grained predictions of gestational age. On a purpose=collected dataset of 88 babies, experiments show that our approach attains an expected error of 6 days and is three times more accurate than the manual postnatal method (Ballard). Making use of images improves predictions by 30% compared to using weight only. This indicates that even with a very small set of data, our method is a viable candidate for postnatal gestational age estimation in areas were USS is not available.',
    'authors': 'M. T. Torres,  M. F. Valstar,  C. Henry,  C. Ward,  D. Sharkey',
    'description':
      '2017 12th IEEE International Conference on Automatic Face & Gesture Recognition (FG 2017). 2017',
    'doi': '10.1109/FG.2017.19',
    'duplicate': false,
    'email': ['michel.valstar@nottingham.ac.uk'],
    'email_new': ['michel.valstar@nottingham.ac.uk'],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7961726',
    'key': '351c16ad-87b3-462b-a38f-eca2ae400dd6',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title': 'small sample deep learning for newborn gestational age estimation',
    'tools': ['caffe'],
    'year': 2017.0,
    'id': '351c16ad-87b3-462b-a38f-eca2ae400dd6.pdf',
    'use': 1.0,
    'domain': 'gestation',
    'task_1': 'segmentation',
    'task_2': 'regression',
    'data_type': 'photographic',
    'location': 'newborn',
    'method_1': 'fullycnn',
    'method_2': 'cnn',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': 'https://github.com/shelhamer/fcn.berkeleyvision.org',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': '88',
    'training': 0.0,
    'validation': 0.0,
    'testing': 0.0,
    'cross validation': 1.0,
    'country_1': 'UK',
    'country_2': '0',
    'country_3': 0.0,
    'journal': '0',
    'impact_factor': 0.0,
    'conference':
      '2017 IEEE 12th International Conference on Automatic Face & Gesture Recognition',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Automatic localization of the standard plane containing complicated anatomical structures in ultrasound (US) videos remains a challenging problem. In this paper, we present a learning-based approach to locate the fetal abdominal standard plane (FASP) in US videos by constructing a domain transferred deep convolutional neural network (CNN). Compared with previous works based on low-level features, our approach is able to represent the complicated appearance of the FASP and hence achieve better classification performance. More importantly, in order to reduce the overfitting problem caused by the small amount of training samples, we propose a transfer learning strategy, which transfers the knowledge in the low layers of a base CNN trained from a large database of natural images to our task-specific CNN. Extensive experiments demonstrate that our approach outperforms the state-of-the-art method for the FASP localization as well as the CNN only trained on the limited US training samples. The proposed approach can be easily extended to other similar medical image computing problems, which often suffer from the insufficient training samples when exploiting the deep CNN to represent high-level features.',
    'authors':
      'H. Chen,  D. Ni,  J. Qin,  S. Li,  X. Yang,  T. Wang,  P. A. Heng',
    'description': 'IEEE Journal of Biomedical and Health Informatics. 2015',
    'doi': '10.1109/JBHI.2015.2425041',
    'duplicate': false,
    'email': [
      'jackie.haochen@gmail.com',
      'nidong@szu.edu.cn',
      'jqin@szu.edu.cn',
      'lishengli63@126.com',
      'pheng@cse.cuhk.edu.hk',
    ],
    'email_new': [
      'jackie.haochen@gmail.com',
      'nidong@szu.edu.cn',
      'jqin@szu.edu.cn',
      'lishengli63@126.com',
      'pheng@cse.cuhk.edu.hk',
    ],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7090943',
    'key': '2f4ae966-7c74-4fb8-923e-301ac4d845fa',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title':
      'standard plane localization in fetal ultrasound via domain transferred deep neural networks',
    'tools': ['caffe'],
    'year': 2015.0,
    'id': '2f4ae966-7c74-4fb8-923e-301ac4d845fa.pdf',
    'use': 1.0,
    'domain': 'radiology',
    'task_1': 'plane detection',
    'task_2': '0',
    'data_type': 'us',
    'location': 'fetal',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1':
      'https://github.com/BVLC/caffe/tree/master/models/bvlc_reference_caffenet',
    'based_on_2': '0',
    'transfer_learning': 1.0,
    'total': '519',
    'training': 300.0,
    'validation': 0.0,
    'testing': 219.0,
    'cross validation': 0.0,
    'country_1': 'China',
    'country_2': 'Hong Kong',
    'country_3': 0.0,
    'journal': 'IEEE Journal of Biomedical and Health Informatics',
    'impact_factor': 3.451,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'The purpose of this study is to evaluate transfer learning with deep convolutional neural networks for the classification of abdominal ultrasound images. Grayscale images from 185 consecutive clinical abdominal ultrasound studies were categorized into 11 categories based on the text annotation specified by the technologist for the image. Cropped images were rescaled to 256\u2009\u00d7\u2009256 resolution and randomized, with 4094 images from 136 studies constituting the training set, and 1423 images from 49 studies constituting the test set. The fully connected layers of two convolutional neural networks based on CaffeNet and VGGNet, previously trained on the 2012 Large Scale Visual Recognition Challenge data set, were retrained on the training set. Weights in the convolutional layers of each network were frozen to serve as fixed feature extractors. Accuracy on the test set was evaluated for each network. A radiologist experienced in abdominal ultrasound also independently classified the images in the test set into the same 11 categories. The CaffeNet network classified 77.3% of the test set images accurately (1100/1423 images), with a top-2 accuracy of 90.4% (1287/1423 images). The larger VGGNet network classified 77.9% of the test set accurately (1109/1423 images), with a top-2 accuracy of VGGNet was 89.7% (1276/1423 images). The radiologist classified 71.7% of the test set images correctly (1020/1423 images). The differences in classification accuracies between both neural networks and the radiologist were statistically significant (p\u2009&lt;\u20090.001). The results demonstrate that transfer learning with convolutional neural networks may be used to construct effective classifiers for abdominal ultrasound images.',
    'authors': 'Cheng PM, Malhi HS.',
    'description': 'J Digit Imaging.  2017',
    'doi': null,
    'duplicate': false,
    'email': ['phillip.cheng@med.usc.edu'],
    'email_new': ['phillip.cheng@med.usc.edu'],
    'fullUrl': 'https://www.ncbi.nlm.nih.gov//pubmed/27896451',
    'key': '0dffbac9-2cdf-45f3-9199-dfdeb51b6b2f',
    'keyok': 1,
    'keywords':
      'Artificial neural networks; Classification; Deep learning; Digital image processing; Machine learning',
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'pubmed',
    'title':
      'transfer learning with convolutional neural networks for classification of abdominal ultrasound images',
    'tools': ['caffe'],
    'year': 2017.0,
    'id': '0dffbac9-2cdf-45f3-9199-dfdeb51b6b2f.pdf',
    'use': 1.0,
    'domain': 'radiology',
    'task_1': 'classification',
    'task_2': '0',
    'data_type': 'us',
    'location': 'abdomin',
    'method_1': 'cnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1':
      'https://github.com/BVLC/caffe/tree/master/models/bvlc_reference_caffenet',
    'based_on_2': 'https://gist.github.com/ksimonyan/211839e770f7b538e2d8',
    'transfer_learning': 1.0,
    'total': '185',
    'training': 136.0,
    'validation': 0.0,
    'testing': 49.0,
    'cross validation': 0.0,
    'country_1': 'USA',
    'country_2': '0',
    'country_3': 0.0,
    'journal': 'Journal of Digital Imaging',
    'impact_factor': 1.407,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Ultrasound (US) imaging is a widely used screening tool for obstetric examination and diagnosis. Accurate acquisition of fetal standard planes with key anatomical structures is very crucial for substantial biometric measurement and diagnosis. However, the standard plane acquisition is a labor-intensive task and requires operator equipped with a thorough knowledge of fetal anatomy. Therefore, automatic approaches are highly demanded in clinical practice to alleviate the workload and boost the examination efficiency. The automatic detection of standard planes from US videos remains a challenging problem due to the high intraclass and low interclass variations of standard planes, and the relatively low image quality. Unlike previous studies which were specifically designed for individual anatomical standard planes, respectively, we present a general framework for the automatic identification of different standard planes from US videos. Distinct from conventional way that devises hand-crafted visual features for detection, our framework explores in- and between-plane feature learning with a novel composite framework of the convolutional and recurrent neural networks. To further address the issue of limited training data, a multitask learning framework is implemented to exploit common knowledge across detection tasks of distinctive standard planes for the augmentation of feature learning. Extensive experiments have been conducted on hundreds of US fetus videos to corroborate the better efficacy of the proposed framework on the difficult standard plane detection problem.',
    'authors':
      'H. Chen,  L. Wu,  Q. Dou,  J. Qin,  S. Li,  J. Z. Cheng,  D. Ni,  P. A. Heng',
    'description': 'IEEE Transactions on Cybernetics. 2017',
    'doi': '10.1109/TCYB.2017.2685080',
    'duplicate': false,
    'email': [
      'jackie.haochen@gmail.com',
      'jzcheng@szu.edu.cn',
      'nidong@szu.edu.cn',
    ],
    'email_new': [
      'jackie.haochen@gmail.com',
      'jzcheng@szu.edu.cn',
      'nidong@szu.edu.cn',
    ],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7890445',
    'key': '6240cc26-0b20-4dc5-ae04-1085c9723f53',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title':
      'ultrasound standard plane detection using a composite neural network framework',
    'tools': ['caffe'],
    'year': 2017.0,
    'id': '6240cc26-0b20-4dc5-ae04-1085c9723f53.pdf',
    'use': 1.0,
    'domain': 'radiology',
    'task_1': 'plane detection',
    'task_2': '0',
    'data_type': 'us',
    'location': 'fetal',
    'method_1': 'cnn',
    'method_2': 'rnn',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 0.0,
    'data_shared': 0.0,
    'data_link_1': '0',
    'data_link_2': '0',
    'data_ok': 0.0,
    'code_public': 0.0,
    'code_link': '0',
    'code_ok': 0.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 1.0,
    'total': null,
    'training': null,
    'validation': null,
    'testing': null,
    'cross validation': 0.0,
    'country_1': 'China',
    'country_2': 'HongKong',
    'country_3': 0.0,
    'journal': 'IEEE Transactions on Cybernetics',
    'impact_factor': 7.384,
    'conference': '0',
    'public': 0.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Convolutional Neural Networks (CNNs) have been recently employed to solve problems from both the computer vision and medical image analysis fields. Despite their popularity, most approaches are only able to process 2D images while most medical data used in clinical practice consists of 3D volumes. In this work we propose an approach to 3D image segmentation based on a volumetric, fully convolutional, neural network. Our CNN is trained end-to-end on MRI volumes depicting prostate, and learns to predict segmentation for the whole volume at once. We introduce a novel objective function, that we optimise during training, based on Dice coefficient. In this way we can deal with situations where there is a strong imbalance between the number of foreground and background voxels. To cope with the limited number of annotated volumes available for training, we augment the data applying random non-linear transformations and histogram matching. We show in our experimental evaluation that our approach achieves good performances on challenging test data while requiring only a fraction of the processing time needed by other previous methods.',
    'authors': 'F. Milletari,  N. Navab,  S. A. Ahmadi',
    'description':
      '2016 Fourth International Conference on 3D Vision (3DV). 2016',
    'doi': '10.1109/3DV.2016.79',
    'duplicate': false,
    'email': [
      'fausto.milletari@tum.de',
      'navab@cs.tum.edu',
      'ahmad.ahmadi@med.uni-muenchen.de',
    ],
    'email_new': [
      'fausto.milletari@tum.de',
      'navab@cs.tum.edu',
      'ahmad.ahmadi@med.uni-muenchen.de',
    ],
    'fullUrl': 'http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7785132',
    'key': '4c992cf9-376f-4048-977a-2eba0a611c86',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': null,
    'phase': 1,
    'received': 0,
    'source': 'ieee',
    'title':
      'v-net: fully convolutional neural networks for volumetric medical image segmentation',
    'tools': ['caffe'],
    'year': 2016.0,
    'id': '4c992cf9-376f-4048-977a-2eba0a611c86.pdf',
    'use': 1.0,
    'domain': 'radiology',
    'task_1': 'segmentation',
    'task_2': '0',
    'data_type': 'mri',
    'location': 'prostate',
    'method_1': 'fullycnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 1.0,
    'data_shared': 0.0,
    'data_link_1': 'https://promise12.grand-challenge.org/download/',
    'data_link_2': '0',
    'data_ok': 1.0,
    'code_public': 1.0,
    'code_link': 'https://github.com/faustomilletari/VNet',
    'code_ok': 1.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': null,
    'training': null,
    'validation': null,
    'testing': null,
    'cross validation': 0.0,
    'country_1': 'Germany',
    'country_2': '0',
    'country_3': 0.0,
    'journal': '0',
    'impact_factor': 0.0,
    'conference': '2016 Fourth International Conference on 3D Vision',
    'public': 1.0,
    'Unnamed: 35': null,
  },
  {
    'abstract':
      'Recently deep residual learning with residual units for training very deep neural networks advanced the state-of-the-art performance on 2D image recognition tasks, e.g., object detection and segmentation. However, how to fully leverage contextual representations for recognition tasks from volumetric data has not been well studied, especially in the field of medical image computing, where a majority of image modalities are in volumetric format. In this paper we explore the deep residual learning on the task of volumetric brain segmentation. There are at least two main contributions in our work. First, we propose a deep voxelwise residual network, referred as VoxResNet, which borrows the spirit of deep residual learning in 2D image recognition tasks, and is extended into a 3D variant for handling volumetric data. Second, an auto-context version of VoxResNet is proposed by seamlessly integrating the low-level image appearance features, implicit shape information and high-level context together for further improving the volumetric segmentation performance. Extensive experiments on the challenging benchmark of brain segmentation from magnetic resonance (MR) images corroborated the efficacy of our proposed method in dealing with volumetric data. We believe this work unravels the potential of 3D deep learning to advance the recognition performance on volumetric image segmentation.',
    'authors': 'Hao Chen, Qi Dou, Lequan Yu, Pheng-Ann Heng',
    'description': 'Retrieved from arXiv. 2016',
    'doi': null,
    'duplicate': false,
    'email': ['hchen@cse.cuhk.edu.hk'],
    'email_new': ['hchen@cse.cuhk.edu.hk'],
    'fullUrl': 'https://arxiv.org/abs/1608.05895',
    'key': '6054d13e-6f9a-406f-a4d8-1f2d4be89100',
    'keyok': 1,
    'keywords': null,
    'pdfUrl': 'https://arxiv.org/pdf/1608.05895.pdf',
    'phase': 1,
    'received': 0,
    'source': 'arxiv',
    'title':
      'voxresnet: deep voxelwise residual networks for volumetric brain segmentation',
    'tools': ['caffe'],
    'year': 2016.0,
    'id': '6054d13e-6f9a-406f-a4d8-1f2d4be89100.pdf',
    'use': 1.0,
    'domain': 'radiology',
    'task_1': 'segmentation',
    'task_2': '0',
    'data_type': 'mri',
    'location': 'brain',
    'method_1': 'rcnn',
    'method_2': '0',
    'method_3': '0',
    'data_public': 0.0,
    'data_upon_request': 1.0,
    'data_shared': 0.0,
    'data_link_1': 'http://mrbrains13.isi.uu.nl/',
    'data_link_2': '0',
    'data_ok': 1.0,
    'code_public': 1.0,
    'code_link': 'http://appsrv.cse.cuhk.edu.hk/~hchen/research/seg_brain.html',
    'code_ok': 1.0,
    'pseudocode': 1.0,
    'based_on_1': '0',
    'based_on_2': '0',
    'transfer_learning': 0.0,
    'total': null,
    'training': null,
    'validation': null,
    'testing': null,
    'cross validation': 1.0,
    'country_1': 'Hong Kong',
    'country_2': '0',
    'country_3': 0.0,
    'journal': 'NeuroImage',
    'impact_factor': 5.835,
    'conference': '0',
    'public': 1.0,
    'Unnamed: 35': null,
  },
];
export default data;
